chain/actors/builtin/init/actor.go.template

```diff
@@ -87,6 +87,28 @@ type State interface {
 	// Sets the address map for the init actor. This should only be used for testing.
 	SetAddressMap(mcid cid.Cid) error
 
-	AddressMap() (adt.Map, error)
 	GetState() interface{}
+
+	Code() cid.Cid
```

Comment:
Please add these for all the sad unwanted actors that you haven't touched.

Reply:
done via: [a78b642](https://github.com/filecoin-project/lotus/pull/9126/commits/a78b64202ecb359d9144deb7a310cfdf3ec13277), [6d04195](https://github.com/filecoin-project/lotus/pull/9126/commits/6d041958106af625a8edb4ec9fcacbc11b1afa02), [9d0a2e6](https://github.com/filecoin-project/lotus/pull/9126/commits/9d0a2e673e7d8f2160bb38827fb21152f324c328), [9b7af83](https://github.com/filecoin-project/lotus/pull/9126/commits/9b7af83fa909a444d923814ce9abf6e44c7bffff)